src/main/java/com/mercadolibre/bootcampmelifrescos/exceptions/ApiException.java

```diff
@@ -0,0 +1,56 @@
+package com.mercadolibre.bootcampmelifrescos.exceptions;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Getter
+@Setter
+public class ApiException extends Exception{
+    private static final long serialVersionUID = 1L;
+
+    private String code;
+    private String description;
+    private Integer statusCode = HttpStatus.INTERNAL_SERVER_ERROR.value();;
```

Comment:
```suggestion
    private Integer statusCode = HttpStatus.INTERNAL_SERVER_ERROR.value();
```

Ficou com um ponto e vírgula a mais, não?